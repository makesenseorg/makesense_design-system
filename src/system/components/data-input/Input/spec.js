import { shallowMount } from '@vue/test-utils'
import Comp from './Input.vue'

describe('Input.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Comp)
    expect(wrapper.element).toMatchSnapshot()
  })
})