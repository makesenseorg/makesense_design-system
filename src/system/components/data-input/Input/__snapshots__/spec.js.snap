// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input.vue matches snapshot 1`] = `
<ds-form-item-stub>
  <div
    class="ds-input-wrap"
  >
    <!---->
     
    <input
      class="ds-input"
      tabindex="0"
      type="text"
    />
     
    <!---->
  </div>
</ds-form-item-stub>
`;
